web/modules/custom/pineapple_mail/pineapple_mail.module

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @file
+ * Contains pineapple_mail.module.
+ */
+
+use Drupal\Component\Utility\Html;
+
+/**
+ * Implements hook_mail().
+ */
+function pineapple_mail_mail($key, &$message, $params) {
+  $options = array(
+    'langcode' => $message['langcode'],
+  );
+  switch ($key) {
+    case 'node_insert':
+      $message['from'] = \Drupal::config('system.site')->get('mail');
+      $message['subject'] = t('Subject: @title', array('@title' => $params['title']), $options);
+      $message['body'][] = Html::escape($params['message']);
+      break;
+  }
+}
+
+function pineapple_mail_send_mail(Drupal\Core\Entity\EntityInterface $entity) {
+  $mailManager = \Drupal::service('plugin.manager.mail');
+  $module = 'pineapple_mail';
+  $key = 'node_insert';
+  $to = \Drupal::config('system.site')->get('mail');
+  $entity_link = $entity->toUrl()->toString();
+  $entity_title = $entity->label();
+  $host = \Drupal::request()->getSchemeAndHttpHost();
+  $params['message'] = 'The user ' .\Drupal::currentUser()->getAccountName(). ' indicated that ' .$entity_title. ' is ready for review. View it here: ' .$host. $entity_link;
+  $params['title'] = $entity_title . ' is ready to review';
+  $langcode = \Drupal::currentUser()->getPreferredLangcode();
+  $send = true;
+
+  $result = $mailManager->mail($module, $key, $to, $langcode, $params, NULL, $send);
+  if ($result['result'] !== true) {
+    $message = 'There was a problem sending your message and it was not sent. If this issue persists, contact the site administrator.';
+    \Drupal::messenger()->addMessage($message);
+    \Drupal::logger('mail-log')->error($message);
+  }
+  else {
+    $message = 'Your content has been flagged as ready for review and will be reviewed and published by the site administrator.';
+    \Drupal::messenger()->addMessage($message);
+    \Drupal::logger('mail-log')->notice($message);
+  }
+}
+
+/**
+ * Implements hook_ENTITY_TYPE_presave().
+ */
+function pineapple_mail_node_presave(Drupal\Core\Entity\EntityInterface $entity) {
```

Comment:
Currently the code in this is running for any node. It may throw notices if it runs on a node where the field referenced doesn't exist. So I'd wrap the whole thing in an if statement that checks if the node is article or recipe.

Reply:
Of course! Done